### PR TITLE
Add Downscale augmentation

### PR DESCRIPTION
**Pull request by arsenyinfo**

### CODE REVIEW

File: albumentations/augmentations/transforms.py [hunk 1]
```diff
@@ -2572,6 +2572,42 @@ def get_transform_init_args_names(self):
...
+        assert scale_max < 1, "Expected scale_max to be less than 1, got {}".format(scale_max)
+        self.scale_min = scale_min
+        self.scale_max = scale_max
+        self.interpolation = interpolation
+
+    def apply(self, image, scale, interpolation, **params):
+        return F.downscale(image, scale=scale, interpolation=interpolation)
+
+    def get_params(self):
+        return {'scale': np.random.rand() * (self.scale_max - self.scale_min) + self.scale_min,
```
Comment (Dipet) [inline albumentations/augmentations/transforms.py]: You can use `random.uniform(self.scale_min, self.scale_max)`

Comment (arsenyinfo) [reply albumentations/augmentations/transforms.py]: thnx, fixed

**Comment by arsenyinfo** [comment]
🎉 the build is finally green

**Comment by arsenyinfo** [comment]
close due to harsh merge conflicts

**Pull request closed without merging**